WordPress/gutenberg
# Spacing control: Replace sides dropdwon with link button

## What?


This changes removes sides dropdown from the `SpacingSizesControl` (Padding and Margin), and replaces it with a link button.

Fixed #63963 (partially)

Note: no updates to icons in this iteration.

## Testing Instructions





* Choose a block such as gallery (with spacing settings enabled) and observe that sides dropdown is now replaced with link button.
* Link should toggle between `horizontal & vertical` view and `all sides` custom view.
* It should be the same behaviour in the global style panel

### Testing Instructions for Keyboard


## Screenshots or screencast 

<img width="285" alt="image" src="https://github.com/user-attachments/assets/a13a2f48-6a4b-4b0e-96e6-b72926f014ee">
<img width="285" alt="image" src="https://github.com/user-attachments/assets/7863db28-6afc-4914-b789-612fb767f138">
